docs/getting-started/community-membership.sh

```diff
@@ -0,0 +1,119 @@
+| [Owner](#owner) | Sponsor from 3 owners, helps drive the overall KubeEdge project | Drive the overall technical roadmap of the project and set priorities of activities in release planning | KubeEdge GitHub organization Admin access |
+
+
+**Note :** It is mandatory for all KubeEdge community members to follow KubeEdge [Code of Conduct](https://github.com/kubeedge/kubeedge/blob/master/CODE_OF_CONDUCT.md).
+
+## Member
+
+Members are active participants in the community who contribute by authoring PRs,
+reviewing issues/PRs or participate in community discussions on slack/mailing list. 
+
+### Privileges
+
+- Member of the Kubernetes GitHub organization
+- Can be assigned to issues and PRs and community members can also request their review
+
+### Requirements
+
+- Sponsor from 2 approvers
+- Actively contributed to the community. Contributions may include, but are not limited to:
+    - Authoring PRs
+    - Reviewing issues/PRs authored by other community members
+    - Participating in community discussions on slack/mailing list
+    - Participate in KubeEdge community meetings
+
+### Responsibilities 
+
+- Participate in assigned issues and PRs
+- Welcome new contributors
+- Guide new contributors to relevant docs/files
+- Help/Motivate new members in contributing to KubeEdge
+
+## Approver
+
+Approvers are active members who have good experience and knowledge of the domain.
+They have actively participated in the issue/PR reviews and have identified relevant issues during review. 
+
+
+### Requirements
+
+- Sponsor from 2 maintainers
+- Member for at least 2 months
+- Have reviewed good number of PRs
+- Have good codebase knowledge
+
+
+### Responsibilities and Privileges
+
+- Review code to maintain/improve code quality
+- Acknowledge and work on review requests from community members
+- May approve code contributions for acceptance related to relevant expertise
+- Continue to contribute and guide other community members to contribute in KubeEdge project
+
+## Maintainer
+
+Maintainers are approvers who have shown good technical judgement in feature design/development in the past.
+Has overall knowledge of the project and features in the project.
+
+
+### Requirements
+
+- Sponsor from 2 owners
+- Approver for at least 2 months
+- Nominated by a project owner
+- Good technical judgement in feature design/development
+
+### Responsibilities and privileges
+
+- Participate in release planning
+- Maintain project code quality
+- Ensure API compatibility with forward/backward versions based on feature graduation criteria
+- Analyze and propose new features/enhancements in KubeEdge project
+- Demonstrate sound technical judgement
+- Mentor contributors and approvers
+- Have write access to relevant repository (able to Merge PRs) 
```

Comment:
I am curious to learn if Approvers are able to merge PRs?

Reply:
Since we not yet enabled bot merging code, we would probably only have the maintainers able to click the merge button